weave/trace_server/opentelemetry/constants.py

```diff
@@ -40,3 +40,9 @@
     ],
     "model_parameters": ["gen_ai.request", "llm.invocation_parameters"],
 }
+
+# Wandb/Weave specific call attributes
+WB_KEYS = {
+    "display_name": ["wandb.display_name"],
+    "project_id": ["wandb.project_id"],
```

Comment:
this seems like a potential auth issue. cannot blindly trust this

Reply:
project_id is totally unused at the moment. I added it here because it's related to a followup PR I am working through.

Reply:
please remove this

Reply:
removed in https://github.com/wandb/weave/tree/zach/no-synthetic-io-keys